app.js

```diff
@@ -221,3 +224,23 @@ submit.addEventListener('submit', (e) => {
   return true;
 });
 
+function savaData() {
+  const userData = {
+    username: fullname.value,
+    email: email.value,
+    message: message.value,
+  };
+  localStorage.setItem('data', JSON.stringify(userData));
+}
+
+fullname.addEventListener('focusout', savaData);
+email.addEventListener('focusout', savaData);
+message.addEventListener('focusout', savaData);
```

Comment:
Rather than using the `focusout` method please consider using `onChange` as currently, it does not save data to local storage until someone focuses out manually 💯 It will help you meet the project requirements.

Reply:
@ABDULALI3468 
Dear reviewer:
If i am changing the focusout to onChange it is not saving my info in the local storage so what should I do?
Please reply fast